### ISSUE TITLE
Testing Question Tagger 2

### ISSUE THREAD

**Issue by abhijeetps** [ameerthehacker/probot-question-tagger-demo#3]
It does really nice job! 🎉

**Comment by abhijeetps** [comment]
I guess, it failed here. Possibly, NLP library that you are using is not working fine. 
Or probably code behind it.
Nice and useful application, by the way, starred your repo.

**Comment by itaditya** [comment]
> Possibly, NLP library that you are using is not working fine.

The current implementation is scanning for keywords like why, how, **does**, etc. that's why it detected your issue description as a comment.

**Comment by abhijeetps** [comment]
I later checked that he isn't using any NLP library.
In slack, he started this discussion by creating an issue and discussing of using NLP Library to identy questions. I thought, he would have used the same, but he didn't as of now. 😄 
Possibly, he will work on it now.

**Comment by ameerthehacker** [comment]
@itaditya  yes you are right I'm not currently using any NLP library and I created it just as an experiment without NLP. As @aps120797  says I will be integrating NLP soon into this making it better and let everyone know.
